flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
so you can remove this and fix the binary only python package?

Reply:
No as we provide 64bit images only in Atoms

Reply:
> so you can remove this and fix the binary only python package?

No as we provide 64bit images only in Atoms

Reply:
aarch64 is 64-bits too.

Reply:
*our images are x86_64 only, I don't think these will works on aarch64